ibis/backends/datafusion/tests/test_client.py

```diff
@@ -0,0 +1,39 @@
+import pyarrow as pa
+import pytest
+
+from .. import Backend
+
+pytestmark = pytest.mark.datafusion
+
+
+@pytest.fixture
+def client(data_directory):
```

Comment:
I've seen this in other backends, but isn't it enough `TestConf.connect`?

Reply:
It is, updated.